t/test.lua

```diff
@@ -196,6 +196,10 @@ function issue_120(request)
   return request
 end
 
+function skipping_test(request)
+    return request
+end
```

Comment:
Since `tnt_pass_http_request` is off, it is not a request object, just first received parameter. I would name and implement it so:

```lua
function echo(...)
    return ...
end
```

It seems, nginx configuration already have `/echo` endpoint, so the separate `/skipping_test` is not necessary. `tnt_http_methods` has POST by default, so we can just use it.

Reply:
That's minor, I just did it myself.